readthedocs/oauth/notifications.py

```diff
@@ -96,5 +99,21 @@
         ),
         type=ERROR,
     ),
+    Message(
+        id=MESSAGE_OAUTH_DEPLOY_KEY_ATTACHED_FAILED_ON_MANUAL_IMPORTED_PROJECT,
+        header=_("Failed to add deploy key to project"),
+        body=_(
+            textwrap.dedent(
+                """
+                We were unable to find this repository in your {{ provider_name }} account.
+                If this is a private repository, you'll need to
+                <a href="https://docs.readthedocs.io/page/guides/importing-private-repositories.html">
+                  add the deploy key manually
+                </a>.
+                """
+            ).strip(),
+        ),
+        type=WARNING,
```

Comment:
I think this would be best as an info message, instead of warning message

```suggestion
        type=INFO,
```